tests/test_wirc_pipeline.py

```diff
@@ -39,16 +39,16 @@
 }
 
 
-def get_cal_path(name: str) -> str:
+def get_cal_path(images: ImageBatch) -> Path:
     """
     Function to get cal path
     Args:
-        name:
+        images:
 
     Returns:
 
     """
-    return os.path.join(test_data_dir, f"wirc/cals/test_{name}.fits")
+    return Path(test_data_dir).joinpath("wirc/cals/test_dark.fits")
```

Comment:
why is it "test_dark" always? If you can't use it for any other cal, change the function name